graphql/resolvers.go

```diff
@@ -359,6 +360,44 @@ func (r *queryResolver) TaskFiles(ctx context.Context, taskID string) ([]*Groupe
 	}
 	return groupedFilesList, nil
 }
+func (r *queryResolver) TaskLogs(ctx context.Context, taskID string) (*RecentTaskLogs, error) {
+	const LogMessageCount = 100
+	var loggedEvents []event.EventLogEntry
+	// loggedEvents is ordered ts descending
+	loggedEvents, err := event.Find(event.AllLogCollection, event.MostRecentTaskEvents(taskID, LogMessageCount))
+	if err != nil {
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("Unable to find EventLogs for task %s: %s", taskID, err.Error()))
+	}
+
+	// remove all scheduled events except the youngest and push to filteredEvents
```

Comment:
I believe this is the weird thing we do where we try to collapse "scheduled" events right? I think it would actually be better to display all events here, and open a ticket for the server to not log that many scheduled events

Reply:
https://jira.mongodb.org/browse/EVG-7497